rest-client-quickstart/src/main/resources/application.properties

```diff
@@ -1,8 +1,11 @@
 %prod.quarkus.datasource.db-kind=postgresql
-%prod.quarkus.datasource.username=quarkus
-%prod.quarkus.datasource.password=quarkus
-%prod.quarkus.datasource.jdbc.url=jdbc:postgresql:security_jpaTODO
+%prod.quarkus.datasource.username=postgres
+%prod.quarkus.datasource.password=Yourstrongpassword.
+%prod.quarkus.datasource.jdbc.url=jdbc:postgresql:///postgres?cloudSqlInstance=cs-devops-62582-group8:europe-north1:quarkus&socketFactory=com.google.cloud.sql.postgres.SocketFactory
```

Comment:
Move to environment variable

Reply:
Just move the"cs-devops-62582-group8:europe-north1:quarkus" part to env var. The rest can stay

Reply:
Where should the `.env` file be located, i am a bit lost, did some reading at https://quarkus.io/guides/config-reference#env-file but brain not good.

Reply:
I was thinking you do the same as with the pem keys. Pass them as flags when launching the gcloud server

Reply:
Like this

https://github.com/Complex-systems-and-devops-gruppe-8/backend/blob/b663c1e75bd7805af6ccb3526843720fe46f8ecd/.github/workflows/prod.yml#L80